math/strobogrammatic_number.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file
+ * @brief Program to check if a number is a strobogrammatic number in the decimal system.
```

Comment:
could you add a link to the source thanks!!! ❤️

Reply:
done!